types/office-js-preview/index.d.ts

```diff
@@ -8907,9 +8907,11 @@ declare namespace Office {
             /**
              * The attachment is stored in a cloud location, such as OneDrive.
              *
-             * **Important**: In Read mode, the `id` property of the attachment's {@link Office.AttachmentDetails | details} object
+             * **Important**: In Read mode, the `id` property of the attachment's
+             * {@link https://learn.microsoft.com/javascript/api/outlook/office.attachmentdetails | AttachmentDetails} object
```

Comment:
JFYI - in the preview version, you can continue to use these objects directly instead of the learn links since all the requirement sets are available. However, I can appreciate your switching them for consistency with the office-js version